### PR TITLE
Docs: Update component/selector EN docs.

### PR DESCRIPTION
**Pull request by Mo3et**
Update  EN `component/selector.md` to sync with CN docs && correct some styles.
Solve  #184.

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md [hunk 1]
```diff
@@ -14,72 +14,70 @@ keywords:
...
 
-// Create a route Filter: filter instances with version number "2.0.0"
+// Create a route Filter: filter instances with version number "2.0.0".
 filter := filter.Version("2.0.0")
+// Due to the limitations of the gRPC framework, only the global balancer name can be used to inject Selector.
+selector.SetGlobalSelector(wrr.NewBuilder)
 
 conn, err := grpc.DialInsecure(
   context.Background(),
   grpc.WithEndpoint("discovery:///helloworld"),
```
Comment (Mo3et) [inline i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md:77]: At the same time I have a question, The `///` in `discovery:///helloworld` is wrong or have  a meaning.  
I would appreciate it if  you told me this.

Comment (Windfarer) [reply i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md:77]: you may refer to https://pkg.go.dev/google.golang.org/grpc/resolver#Target

**Review by Windfarer — APPROVED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md (hunk 1 above)
Comment (Mo3et) [reply i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md:77]: Thanks.

Comment (Mo3et) [reply i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md:77]: > you may refer to https://pkg.go.dev/google.golang.org/grpc/resolver#Target

Thanks.

**Pull request merged**